src/i_video.c

```diff
@@ -1452,6 +1454,11 @@ static void I_InitGraphicsMode(void)
 
     I_InitWindowIcon();
 
+    if (fullscreen && !exclusive_fullscreen)
+    {
+        SDL_SetWindowGrab(screen, SDL_TRUE);
+    }
+
```

Comment:
It currently keeps the mouse inside when using fullscreen, but the mouse can still move out in exclusive fullscreen.